opentelemetry-otlp/examples/basic-otlp-http/src/hyper.rs

```diff
@@ -0,0 +1,45 @@
+use async_trait::async_trait;
```

Comment:
Do you think we can instead update the [HttpConfig ](https://github.com/open-telemetry/opentelemetry-rust/blob/e0fb7fe3b7ab343cdf2f702b04d04976dc5542ca/opentelemetry-otlp/src/exporter/http/mod.rs#L53) in opentelemetry-otlp with the hyper client, and so not have the special handling here